src/tutorials/Blog/01.vue

```diff
@@ -38,16 +36,11 @@ const _solution = `
 /* globals ipfs */
 
 const run = async () => {
-  const natCid = await ipfs.dag.put({author: "Nat"})
-  const samCid = await ipfs.dag.put({author: "Sam"})
-  const treePostCid = await ipfs.dag.put({
-    content: "trees",
-    author: samCid
-  })
-  const computerPostCid = await ipfs.dag.put({
-    content: "computers",
-    author: natCid
-  })
+  const natCid = await ipfs.dag.put({ author: "Nat" })
+  const samCid = await ipfs.dag.put({ author: "Sam" })
+
+  const treePostCid = await ipfs.dag.put({ content: "trees", author: samCid })
```

Comment:
Although the result of the code is the same, I consider it more beginner-friendly and easier to read when you show objects with each property on its own line. Since you'll be enabling solution viewing in a later PR, let's leave this formatting (and the line below) in the way that's easier to read.